src/index.ts

```diff
@@ -22,6 +22,11 @@ export interface Options {
    * @returns {string}
    */
   format?: (emoji: string, jiraUrls: string[]) => string;
+  /**
+   * The location of the JIRA issue, either the PR title, or the git branch
+   * Defaults to `title`
+   */
+  location?: "title" | "branch";
```

Comment:
I'm sort of wondering if we should even make this an option. Part of me wants to make this plugin "smarter" or closer to "zero config" and just look in the following places for a JIRA ticket ID without requiring the user to configure this:

* `danger.github.pr.title` - PR title
* `danger.github.pr.head.ref` - the PR head branch name
* (I think that's it right now)

What do you think about that as someone who uses this plugin? Do you want more configurability, or do you want things to Just Work™?

Reply:
Maybe also check all the commit messages too? Link all the things 🔗

Reply:
Oooh that would be pretty nice actually. What would it do if it found multiple tickets; just spit out links to all of em?

Reply:
In that vein we could remove the need to specify a JIRA key and look for anything that looks like `ABC[DEF]-123`? In the meantime how would you feel about merging this and tackling all of that as a follow up?

Reply:
Yeah this can be a follow up in a future release. This is a great non-breaking addition to the plugin. 🎉